docs/index.md

```diff
@@ -67,7 +67,7 @@ id: home
       <h3>A Component Using External Plugins</h3>
       <p>
         React is flexible and provides hooks that allow you to interface with
-        other libraries and frameworks. This example uses **remarkable**, an
+        other libraries and frameworks. This example uses <strong>remarkable</strong>, an
```

Comment:
The new docs site does not render the bold markdown tags. They have to be converted to `<strong>` explicitly.

Reply:
Not quite accurate. The `index.md` page specifically is just filled with HTML (see [here](https://raw.githubusercontent.com/yangshun/react/4330c6fe44b403811c65cb8da155d5f610457329/docs/index.md)) and Remark doesn't parse markdown syntax when it's within an HTML tag.

We have an issue filed to clean this up in the future (removing inline HTML+CSS from files like this) but for now this change is 👍